RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2241,7 +2241,8 @@ boolean isUnclePAvailable()
 	{
 		return false;
 	}
-	return true;
+	string page_text = visit_url("place.php?whichplace=desertbeach");
```

Comment:
Is this supposed to be future-proofing, or working around a known case/Mafia desync bug? Comment would be great.

Reply:
Now that I got some experience.
I have seen this kind of breakage happen often resulting in infinite loop whenever we just assume success.
just doing
```
visit url / adventure
return true;
```
is just asking for infinite loops whenever you fail for some reason or another. maybe the path has unusual access requirements. maybe state get desynced. but whatever happens this kind of assumption of success reuslts in infinite loops. so its a pretty good idea to use a protective method to only return true if it actually succeeded in doing what you set it to do. (such as visiting a URL or adventuring in a spot)